### ISSUE TITLE
Move the archive

### ISSUE THREAD

**Issue by mcharg** [ONRR/nrrd#368]
I think Jeff came up with a plan in a previous issue, but we need to implement that plan before the hard switchover.

**Comment by jennmalcolm** [comment]
Needs to be done before end of Sep 2020.

**Comment by jennmalcolm** [comment]
archiving whole old site

**Comment by master12** [comment]
deployed to the dev space as a test.

https://dev-doi-extractives-data.app.cloud.gov/

**Comment by mcharg** [comment]
We should probably remove the beta site banner and put in a banner that says something about this being an archived site.

**Comment by mcharg** [comment]
We could just change the text on the beta banner to something like:

"This is an archived version of this website that is no longer being updated. For current data, visit the [latest version of the site](link to live site)."

**Comment by mcharg** [comment]
We also need to point the link to the archive on the About page to the archive on this site and may also need to setup redirects to any pages that exist on the current site, but not the new site to here.  Should that be a separate issue?
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/ad1ffee4-f7b4-4ab4-a686-ae4435d9d424)

**Comment by master12** [comment]
I am moving this to next sprint.

**Comment by master12** [comment]
https://doi-extractives-data.app.cloud.gov/

Will create a new ticket to request a new domain
